src/MoBi.Presentation/DTO/PathAndValueEntityDTO.cs

```diff
@@ -1,16 +1,55 @@
 using System.Collections.Generic;
 using System.ComponentModel;
+using System.IO;
 using MoBi.Core.Helper;
 using OSPSuite.Core.Domain;
 using OSPSuite.Core.Domain.Builder;
 using OSPSuite.Core.Domain.Formulas;
 using OSPSuite.Core.Domain.UnitSystem;
 using OSPSuite.Utility.Reflection;
 using OSPSuite.Utility.Validation;
+using static MoBi.Assets.AppConstants.Captions;
+using static OSPSuite.Core.Domain.Constants.Distribution;
 
 namespace MoBi.Presentation.DTO
 {
-   public class PathAndValueEntityDTO<T> : BreadCrumbsDTO<T>, IPathAndValueEntityDTO where T : PathAndValueEntity, IValidatable, INotifier, IWithDimension, IWithDisplayUnit, IUsingFormula
+   public class PathAndValueEntityDTO<T, TSubParameter> : PathAndValueEntityDTO<T> where T : PathAndValueEntity where TSubParameter : PathAndValueEntityDTO<T>
+   {
+      private readonly List<TSubParameter> _subParameters = new List<TSubParameter>();
+      public IReadOnlyList<TSubParameter> SubParameters => _subParameters;
+
+      public ObjectPath Path => PathWithValueObject.Path;
+
+      public TSubParameter MeanDTO => SubParameters.FindByName(MEAN);
+
+      protected PathAndValueEntityDTO(T underlyingObject) : base(underlyingObject)
+      {
+      }
+
+      public void AddSubParameter(TSubParameter subParameter)
+      {
+         _subParameters.Add(subParameter);
+      }
+
+      public bool IsDistributed => PathWithValueObject.DistributionType != null;
+
+      public string DistributionType => $"{PathWithValueObject.DistributionType} {Distribution}";
```

Comment:
THis will look weird if it's not Distributed
PathWithValueObject.DistributionType==null

Reply:
I don't understand what the text is going to be. What is Distributiion in this case?

Reply:
oh I see this is the text from constant "Distribution"